### PR TITLE
Add `-q` as alternative to `--quiet` for meson install

### PR DESCRIPTION
**Pull request by jeltz**
Since `meson test` already has both `-q` and `--quiet` it makes a lot of
sense to add the short option to `meson install` too for reasons of
symmetry.

Also changes the output order of `-n` and `--dry-run` for consistency in a separate commit.

### CODE REVIEW

File: data/shell-completions/zsh/_meson [hunk 1]
```diff
@@ -217,7 +217,7 @@ local -a meson_commands=(
     "$__meson_cd"
     '--no-rebuild[Do not rebuild before installing]'
     '--only-changed[Do not overwrite files that are older than the copied file]'
-    '--quiet[Do not print every file that was installed]'
+    '(--quiet -q)'{'--quiet','-q'}'[Do not print every file that was installed]'
```
Comment (eli-schwartz) [inline data/shell-completions/zsh/_meson:220]: Please add this to the bash completion as well.

**Review by eli-schwartz — CHANGES_REQUESTED** [review]

File: data/shell-completions/zsh/_meson (hunk 1 above)
Comment (jeltz) [reply data/shell-completions/zsh/_meson:220]: Thanks, forgot that!

Comment (eli-schwartz) [reply data/shell-completions/zsh/_meson:220]: I think you added this to the wrong commit?

Comment (jeltz) [reply data/shell-completions/zsh/_meson:220]: Oh, damn will fix.

Comment (jeltz) [reply data/shell-completions/zsh/_meson:220]: Fixed!

**Review by eli-schwartz — APPROVED** [review]

**Comment by jeltz** [comment]
Thanks for the quick reviews and merge!

**Pull request merged**